server.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.route("/ping").get((req, res) => {
+  res.send("pong");
+});
+
+app.listen(port, () => {
+  console.log(`App is running on PORT: ${port}`);
+});
+
+module.exports = app
```

Comment:
Add a newline at the end of the file. This is a common convention that helps prevent certain issues when concatenating or processing files. [medium]